src/test/java/marquez/api/resources/NamespaceResourceTest.java

```diff
@@ -115,9 +117,9 @@ public void testGetNamespaceRequestErrorHandling() throws Throwable {
   public void testListNamespaceWithNoResults() throws MarquezServiceException {
     final List<Namespace> existingCoreModelNamespaces = Collections.emptyList();
     NamespaceResource namespaceResource = new NamespaceResource(namespaceService);
-    when(namespaceService.getAll()).thenReturn(existingCoreModelNamespaces);
+    when(namespaceService.getAll(LIMIT, OFFSET)).thenReturn(existingCoreModelNamespaces);
 
-    Response res = namespaceResource.list();
+    Response res = namespaceResource.list(LIMIT, OFFSET);
```

Comment:
nit: we can make this field final.

Reply:
I only wanted to make sure the tests passed. But, yeah, when we refactor the test suite, we can address felids that are set as `final`